composer.json

```diff
@@ -20,12 +20,12 @@
     "require": {
         "php": ">=5.6.0",
         "robmorgan/phinx": "~0.10.3",
-        "cakephp/orm": "^3.6.0",
-        "cakephp/cache": "^3.6.0"
+        "cakephp/orm": "^3.7.0",
+        "cakephp/cache": "^3.7.0"
```

Comment:
Sure this was necssary for require?

Reply:
It's not necessary.

Reply:
seems yes , it's necessary
https://github.com/cakephp/migrations/blob/1e8074ece648e9001117fff50a43c55b35b9de21/src/Command/CacheClear.php#L61

Reply:
`Cache::delete()` exits in 3.6 too.

Reply:
then , we need 3.7 for these :)
https://github.com/cakephp/migrations/pull/378/files
 I know its on core package

Reply:
I don't understand what you are trying to say.

Reply:
I don't understand what you are trying to say.

Reply:
But in this case it really did install "cakephp/cakephp (3.6.0)"
As such you should only have needed to raise require-dev as the tests need a higher version.
I do see the point of not "knowing" if older versions work, so either way it's not really cool..